### DISCUSSION TITLE
docs folder not being treated as documentation?

### DISCUSSION THREAD

**Discussion by jmgirard** [github-linguist/linguist#5391] [Classification] [answered]
Based on https://github.com/github/linguist/blob/master/lib/linguist/documentation.yml, I would have thought that the contents of my `docs` folder would be excluded from linguist stats. But this does not seem to be happening:

https://github.com/jmgirard/circumplex/search?l=html

I added `docs/* linguist-documentation` to a new `.gitattributes` file but I don't think the process has happened yet to update my repo's stats. So I'm not sure this worked yet, but that said, given the above I would think that these files should already have been excluded. Am I misunderstanding?

**Comment by lildude** [comment, marked as answer]
Your `docs` directory is most definitely not contributing towards your stats. The search results are not influenced by Linguist else you'd never be able to search for files Linguist ignores. The reason for your high HTML percentage will be because of the massive HTML files in the `devel` directory in your repo with  https://github.com/jmgirard/circumplex/blob/229ab6f3e2b81c9f49aaf984c999c4bbc2aae686/devel/lavaan_ssm.nb.html being 1.57MB on its own. The other two files are pretty big too.

**Reply by jmgirard** [reply]
Oh, thanks! I should exclude the `devel` directory then.
